django/contrib/auth/management/__init__.py

```diff
@@ -106,7 +106,7 @@ def create_permissions(app_config, verbosity=2, interactive=True, using=DEFAULT_
         if len(perm.name) > permission_name_max_length:
             raise exceptions.ValidationError(
                 "The verbose_name of %s is longer than %s characters" % (
-                    perm.content_type,
+                    perm.content_type.model,
```

Comment:
Wonder if it might be worth to also specify the `app_label` for clarity here.

```python
"The verbose_name of %s.%s is longer than %s characters" % (
    perm.content_type.app_label,
    perm.content_type.model,
    verbose_name_max_length,
)
```